conftest.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2018 Duncan Macleod
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+"""Pytest configuration module
+
+This just adds an extra command line option to specify the waveform
+approximants to use in the tests
+"""
+
+import pytest
+
+from pycbc.waveform import fd_approximants
+
+__author__ = 'Duncan Macleod <duncan.macleod@ligo.org>'
+
+DEFAULT_APPROXIMANTS = [
+    'TaylorF2',
+    'IMRPhenomPv2',
+]
```

Comment:
Could you add SEOBNRv4 to the default list? How long do the test take as is?

Reply:
@vivenr, on my macbook I get the following error: 

```python
ValueError: Approximant SEOBNRv4 not available
```

Is there something special that needs to be done to set up this approximant?